integration-tests/mtom-awt/src/main/java/io/quarkiverse/cxf/it/ws/mtom/awt/server/ImageService.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkiverse.cxf.it.ws.mtom.awt.server;
+
+import javax.jws.WebMethod;
+import javax.jws.WebService;
+import javax.jws.soap.SOAPBinding;
+import javax.xml.ws.soap.MTOM;
+
+@WebService(name = "ImageService", targetNamespace = "https://quarkiverse.github.io/quarkiverse-docs/quarkus-cxf/test/mtom-awt")
+@SOAPBinding(style = SOAPBinding.Style.DOCUMENT, parameterStyle = SOAPBinding.ParameterStyle.BARE)
+@MTOM
+public interface ImageService {
+
+    @WebMethod
+    ImageData downloadImage(String name);
+
+    @WebMethod
+    String uploadImage(ImageData image);
```

Comment:
@llowinge strange enough, I was only able to make it work when I wrapped java.awt.Image inside a ImageData

Reply:
Otherwise it gave you the same error as with camel-quarkus ?

Reply:
Yes, an unexpected element during deserialization